nagesen/src/components/Dashboard.vue

```diff
@@ -6,17 +6,36 @@
     </div>
 
     <h1>ユーザ一覧</h1>
-    <p>課題４－３で対応のため未実装</p>
+    <ul>
+      <li class="sub">ユーザ名</li>
+      <li v-for="(user,index) in registeredUser" :key="index">
+        <div class="name">{{user.displayName}}</div>
+        <div>
+          <button class="wallet" @click="openWalletModal(user)">walletを見る</button>
+          <button class="wallet">送る</button>
+        </div>
+      </li>
+    </ul>
+
+    <wallet-modal v-if="showWallet" @from-child="closeWalletModal"></wallet-modal>
+
   </div>
 </template>
 
 <script>
+import WalletModal from './WalletModal.vue'
+
 export default {
   name: 'Dashboard',
+  components:{
+    WalletModal
+  },
   data(){
     return {
       displayName: '',
-      remaining: ''
+      remaining: '',
+      registeredUser:[],
```

Comment:
細かいですが、配列であれば複数形でも命名が望ましいです。
```suggestion
      registeredUsers:[],
```